Лабораторная работа 1/task_3.py

```diff
@@ -0,0 +1,11 @@
+list_players = ["Маша", "Петя", "Саша", "Оля", "Кирилл", "Коля"]
+
+a=len(list_players) # общее кол-во игроков
+
+b= a//2 # делим на 2 команды
```

Comment:
В случае, когда вы работаете с несложными операциями, можно производить несколько действий в рамках одной строки. Тогда слияние двух этих строк будет выглядеть следующим образом: 
```suggestion
middle_index = len(list_players) // 2 # общее кол-во игроков
```
Обратите внимание на наименование переменных. Необходимо уложить в ее наименование суть хранимой в ней информации